components/builder-core/src/metrics.rs

```diff
@@ -98,22 +112,14 @@ fn receive(rz: SyncSender<()>, rx: Receiver<MetricTuple>) {
 
     loop {
         let (mtyp, mop, mid, mval): MetricTuple = rx.recv().unwrap();
-        debug!("Received metrics tuple: {:?}", (mtyp, mop, mid, mval));
+        debug!("Received metrics tuple: {:?}", (mtyp, mop, &mid, mval));
 
         match client {
             Some(ref mut cli) => {
                 match mtyp {
                     MetricType::Counter => {
                         match mop {
-                            MetricOperation::Increment => cli.incr(mid),
-                            MetricOperation::Decrement => cli.decr(mid),
-                            _ => error!("Unexpected metric operation: {:?}", mop),
-                        }
-                    }
-                    MetricType::Gauge => {
```

Comment:
It would be nice to keep Gauge code in... at least as a sample or comment on how to add it later.

Reply:
Re-implementing it won't be too difficult, we've got Git history to help us, and we weren't using it anywhere anyway, so I opted to remove it as dead code.